third_party/gfxreconstruct/framework/decode/dive_block_data.h

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2025 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Implementing a class to store GFXR file metadata is necessary to support these changes:
+// - Assemble a modified GFXR file quickly with data chunks from the original file and from stored modifications
+
+#ifndef GFXRECON_DECODE_DIVE_BLOCK_DATA_H
+#define GFXRECON_DECODE_DIVE_BLOCK_DATA_H
+
+#include "util/defines.h"
+
+#include <string>
+#include <vector>
+
+static const unsigned int BUFFER_SIZE = 4096;
```

Comment:
i could be wrong, but i did a quick search, it seems that gfxr prefer using `static constexpr ` for consts with `k` as prefix. Also `buffer size` seems to be a bit too generic especially in a header. maybe something like kDiveBlockBufferSize